api/alembic/versions/871c39cf6c26_fuel_type_abbreviation_should_be_unique.py

```diff
@@ -0,0 +1,28 @@
+"""Fuel type abbreviation should be unique
+
+Revision ID: 871c39cf6c26
+Revises: af108f346073
+Create Date: 2022-02-01 13:53:33.761953
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '871c39cf6c26'
+down_revision = 'af108f346073'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # make fuel type abbreviation index, and unique.
+    op.create_index(op.f('ix_fuel_types_abbrev'), 'fuel_types', ['abbrev'], unique=True)
+    op.create_unique_constraint(None, 'planning_weather_stations', ['station_code', 'planning_area_id'])
```

Comment:
Well spotted. The autogenerate went and put that in. I'm pulling down the latest from prod and re-running to figure out. It could be because it didn't have a name, and now alembic is trying to figure out what to do about it.

Reply:
So - I've named the constraint - and added some hacks to make sure our local database and prod are in sync.